Domain/Shipping/Cargo/Itinerary.cs

```diff
@@ -23,7 +23,7 @@ public Itinerary(IList<Leg> legs)
 
         public UnLocode FirstLoadLocation => Legs.First().LoadLocation;
 
-        public VoyageNumber FirstYoyage => Legs.First().Voyage;
```

Comment:
*Yoy* age (age between 18-25) :stuck_out_tongue_closed_eyes:

Reply:
Legs are not going to be empty, comment [here](https://github.com/klimisa/CargoTracker2FSharp/pull/1#discussion_r365144009)